icu4j/main/core/src/main/java/com/ibm/icu/impl/UCharacterProperty.java

```diff
@@ -1444,20 +1460,73 @@ public static final int mergeScriptCodeOrIndex(int scriptX) {
     /*
      * Properties in vector word 2
      * Bits
-     * 31..26   unused since ICU 70 added uemoji.icu;
-     *          in ICU 57..69 stored emoji properties
+     * 31..26   ICU 75: Identifier_Type bit set
+     *          ICU 70..74: unused
+     *          ICU 57..69: emoji properties; moved to uemoji.icu in ICU 70
      * 25..20   Line Break
      * 19..15   Sentence Break
      * 14..10   Word Break
      *  9.. 5   Grapheme Cluster Break
      *  4.. 0   Decomposition Type
      */
-    //ivate static final int PROPS_2_EXTENDED_PICTOGRAPHIC=26;  // ICU 62..69
-    //ivate static final int PROPS_2_EMOJI_COMPONENT = 27;  // ICU 60..69
-    //ivate static final int PROPS_2_EMOJI = 28;  // ICU 57..69
-    //ivate static final int PROPS_2_EMOJI_PRESENTATION = 29;  // ICU 57..69
-    //ivate static final int PROPS_2_EMOJI_MODIFIER = 30;  // ICU 57..69
-    //ivate static final int PROPS_2_EMOJI_MODIFIER_BASE = 31;  // ICU 57..69
+
+    // https://www.unicode.org/reports/tr39/#Identifier_Status_and_Type
+    // The Identifier_Type maps each code point to a *set* of one or more values.
+    // Some can be combined with others, some can only occur alone.
+    // Exclusion & Limited_Use are combinable bits, but cannot occur together.
+    // We use this forbidden combination for enumerated values.
+    // We use 6 bits for all possible combinations.
+    // If more combinable values are added, then we need to use more bits.
+    //
+    // We do not store separate data for Identifier_Status:
+    // We can derive that from the encoded Identifier_Type via a simple range check.
+
+    // vate static final int ID_TYPE_MASK = 0xfc000000;
```

Comment:
`// vate`?

Reply:
Yes :-)

When I port code from C++ to Java, I usually copy all constants. When some are unused, then I comment them out, and often by overwriting the first few characters rather than indenting, so that they line up with adjacent lines.

Like this:
```diff
     private ...
-    private ...
+    // vate ...
```